docs/application/native/guides/multimedia/sound-manager.md

```diff
@@ -192,21 +198,59 @@ To query sound device information:
 
       ```
       static void
-      _sound_device_connection_changed_cb(sound_device_h device, bool is_connected, void* user_data)
+      _sound_device_connection_changed_cb(sound_device_h device, bool is_connected, void *user_data)
       {
           int ret;
           sound_device_type_e type;
 
           if (is_connected) {
               ret = sound_manager_get_device_type(device, &type);
-              if (type == SOUND_DEVICE_BLUETOOTH)
-                  /* Connected sound device type is Bluetooth, handle accordingly */
+              if (type == SOUND_DEVICE_BLUETOOTH_MEDIA)
+                  /* Connected sound device type is Bluetooth Media, handle accordingly */
+              else
+                  /* Handle accordingly */
+          } else {
+              ret = sound_manager_get_device_type(device, &type);
+              if (type == SOUND_DEVICE_BLUETOOTH_MEDIA)
+                  /* Disconnected sound device type is Bluetooth Media, handle accordingly */
+              else
+                  /* Handle accordingly */
+          }
+      }
+      ```
+
+3. To receive a notification whenever the sound device running state changes:
```

Comment:
Suggestion:
Can we rephrase to:
To receive a notification whenever the sound device changes its running state:

Please verify whether the technical meaning is altered.

Reply:
There are two type of device state.
 - Connection state : connected, disconnected.
 - Running state : running, not running(idle).

Both states are changed by user action(connect/disconnect a device and start/stop a stream respectively). Therefore the device state change is of passive, not of active.

From this point of view, is your suggestion still valid?
(if valid, do I need to change "whenever the sound device connection state changes" to "whenever the sound device changes its connection state"?)

Reply:
It looks good to me, thanks.